ui/src/app/views/workspace-share/component.spec.ts

```diff
@@ -141,101 +149,75 @@ describe('WorkspaceShareComponent', () => {
+    expect(addedUserRole.email).toBe(userValuesStub.items[3].email);
+    expect(addedUserRole.fullName).toBe(userValuesStub.items[3].name);
   }));
 
   it('removes users correctly and does not allow self removal', fakeAsync(() => {
-    workspaceSharePage.fixture.componentRef.instance.userEmail
-        = 'sampleuser1@fake-research-aou.org';
-    workspaceSharePage.fixture.componentRef.instance.accessLevel
-        = WorkspaceAccessLevel.OWNER;
+    workspaceSharePage.fixture.componentRef.instance.userEmail =
+        'sampleuser1@fake-research-aou.org';
+    workspaceSharePage.fixture.componentRef.instance.accessLevel = WorkspaceAccessLevel.OWNER;
     workspaceSharePage.readPageData();
     const de = workspaceSharePage.fixture.debugElement;
     de.queryAll(By.css('.remove-button')).forEach((removeButton) => {
       simulateClick(workspaceSharePage.fixture, removeButton);
     });
     workspaceSharePage.readPageData();
-    const roleNamePairs =
-      convertToUserRoleRow(workspaceSharePage.fixture.componentRef.instance.workspace.userRoles);
-    expect(workspaceSharePage.roleNamePairsOnPage)
-        .toEqual(roleNamePairs);
-    expect(workspaceSharePage.roleNamePairsOnPage.length)
-        .toBe(1);
-    expect(workspaceSharePage.roleNamePairsOnPage[0].fullName)
-        .toBe('Sample User1');
-    expect(workspaceSharePage.roleNamePairsOnPage[0].role)
-        .toEqual(WorkspaceAccessLevel.OWNER);
-  }));
-
-  it('validates and allows usernames', fakeAsync(() => {
-    spyOn(TestBed.get(WorkspacesService), 'shareWorkspace')
-      .and.callThrough();
-
-    workspaceSharePage.readPageData();
-    simulateInput(workspaceSharePage.fixture, workspaceSharePage.emailField, 'sampleuser4');
-    workspaceSharePage.fixture.componentRef.instance.setAccess('Writer');
-
-
-    simulateClick(workspaceSharePage.fixture,
-      workspaceSharePage.fixture.debugElement.query(By.css('.add-button')));
-    workspaceSharePage.readPageData();
-    expect(TestBed.get(WorkspacesService).shareWorkspace)
-        .toHaveBeenCalledWith('defaultNamespace', '1', userValuesStub);
-  }));
-
-  it('validates and allows email addresses', fakeAsync(() => {
-    spyOn(TestBed.get(WorkspacesService), 'shareWorkspace')
-      .and.callThrough();
-
-    workspaceSharePage.readPageData();
-    simulateInput(workspaceSharePage.fixture, workspaceSharePage.emailField,
-      'sampleuser4@fake-research-aou.org');
-    workspaceSharePage.fixture.componentRef.instance.setAccess('Writer');
-
-
-    simulateClick(workspaceSharePage.fixture,
-      workspaceSharePage.fixture.debugElement.query(By.css('.add-button')));
-    workspaceSharePage.readPageData();
-    expect(TestBed.get(WorkspacesService).shareWorkspace)
-      .toHaveBeenCalledWith('defaultNamespace', '1', userValuesStub);
+    expect(workspaceSharePage.roleNamePairsOnPage.length).toBe(1);
+    expect(workspaceSharePage.roleNamePairsOnPage[0].fullName).toBe('Sample User1');
```

Comment:
Nit: If we can, I think this would be good to read from `userValuesStub` but I don't think that is blocking here.